e2e_testing/torchscript/elementwise.py

```diff
@@ -310,3 +310,22 @@ def forward(self, x):
 @register_test_case(module_factory=lambda: ElementwiseClampModule())
 def ElementwiseClampModule_basic(module, tu: TestUtils):
     module.forward(tu.rand(3, 5, low=-10, high=10))
+
+
+# ==============================================================================
+class ElementwiseToDtypeModule(torch.nn.Module):
+    def __init__(self):
+        super().__init__()
+
+    @export
+    @annotate_args([
+        None,
+        ([-1, -1], torch.float32, True)
```

Comment:
can we test for more combinations? Like the cases actually need type conversions.

Reply:
Added a test case from f32 to i64.